README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-    <img alt="Keyrod" src="https://i.imgur.com/BYHGStr.png"/>
+    <img alt="Keyrod" src="https://i.imgur.com/yZKRrO0.png"/>
```

Comment:
Can you please set width of Keyrod logo to 300px? Keystorm has logo with width of 300px too. Take a look:
https://raw.githubusercontent.com/the-rocci-project/keystorm/master/README.md